packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyAction.tsx

```diff
@@ -190,7 +328,7 @@ export const GroupPropertyAction = ({
           );
 
           setPropertyName(response.propertyName);
-          setOldPropertyName(response.propertyName);
+          // setOldPropertyName(response.propertyName);
```

Comment:
Is this supposed to be commented out?

Reply:
This was an old code which isn't called anymore due to the changes, hence its now removed.